doc/rados/operations/pools.rst

```diff
@@ -602,7 +602,7 @@ You may set values for the following keys:
 ``recovery_priority``
 
 :Description: When a value is set it will boost the computed reservation priority
-              by this amount. This value should be less than 30.
+              relative to this amount. This value must be in the range -10 to 10.
```

Comment:
we should update `doc/rados/operations/placement-groups.rst` to say that negative values can now be assigned for less important pools.